client/state/data-layer/wpcom/sites/blog-stickers/add/index.js

```diff
@@ -32,6 +33,17 @@ export function receiveBlogStickerAdd( store, action, next, response ) {
 		receiveBlogStickerAddError( store, action, next );
 		return;
 	}
+
+	store.dispatch(
```

Comment:
Ah you sure? We have the whole `store` in `receiveBlogStickerAdd` so I can pass it along to `receiveBlogStickerAddError`.